.travis.yml

```diff
@@ -4,7 +4,7 @@ sudo: false
 language: python
 
 python:
-    - "3.6"
+    - "3.8"
```

Comment:
Plz keep `3.6` and add `3.8`
https://docs.travis-ci.com/user/languages/python/

Reply:
done